documentation/service-rules/lead-provider-users.md

```diff
@@ -1,3 +1,106 @@
+
+## Non-Breaking Changes
+
+A non-breaking change does not break or disrupt existing functionality or behaviour of the API. Providers can continue to consume the data and integrate and apply the change when they are ready.
+
+Non-breaking changes include:
+
+- New endpoints (for example, ECF transfers)
+- Adding optional fields to the request body
+- Adding new optional filters to an endpoint (e.g., a non-required ability to filter by cohort on the get declarations endpoint)
+
+### Example: Introducing `evidence_held` as an Optional Field in the Request Body to Start Declarations for Cohort 2025
+
+Currently, `evidence_held` is not a required field for ECF start declarations:
+
+#### As-Is Start Declaration Request Body
+
+| Name              | Required (Yes/No) | Type   | Description/Possible Values |
+|------------------|----------------|--------|-----------------------------|
+| `participant_id` | Yes            | String | Unique ID of the participant |
+| `declaration_type` | Yes            | String | The declaration type <br> Possible values: `started` |
+| `declaration_date` | Yes            | String | The event declaration date |
+| `course_identifier` | Yes            | String | The type of course the participant is enrolled on <br> Possible values: `ecf-induction`, `ecf-mentor` |
+
+For cohort 2025 and onwards, evidence types will be an optional field that providers can supply when submitting start declarations.
+
+#### To-Be: Introducing a New Optional Field to the Request Body
+
+| Name              | Required (Yes/No) | Type   | Description/Possible Values |
+|------------------|----------------|--------|-----------------------------|
+| `participant_id` | Yes            | String | Unique ID of the participant |
+| `declaration_type` | Yes            | String | The declaration type <br> Possible values: `started` |
+| `declaration_date` | Yes            | String | The event declaration date |
+| `course_identifier` | Yes            | String | The type of course the participant is enrolled on <br> Possible values: `ecf-induction`, `ecf-mentor` |
+| `evidence_held`  | No             | String | The type of evidence the lead provider holds <br> Possible values: `training-event-attended`, `self-study-material-completed`, `other`, `materials-engaged-with-offline` |
+
+This is a non-breaking change because providers are still able to submit start declarations without supplying an evidence types. If the field was mandatory, providers would encounter an error message when submitting requests.
+
+In this instance, providers can update their integrations to submit start declarations and evidence types whenever they are ready and continue to submit start declarations without an evidence type for the time being.
```

Comment:
I'm a bit confused that we're explaining general API rules here as opposed to how the LP API works - whilst this is useful info, I'm not sure it's the rules of how the LP API works? @shahadRah 

Particularly I got confused and checked the current API as I thought evidence had always been there lol.